narwhal/consensus/src/bullshark.rs

```diff
@@ -58,8 +58,126 @@ pub struct Bullshark {
     pub num_sub_dags_per_schedule: u64,
 }
 
-impl ConsensusProtocol for Bullshark {
-    fn process_certificate(
+impl Bullshark {
```

Comment:
To be honest now I see less of a reason to have `Bullshark` as a separate struct and not merged with `Consensus`. Let's catch up whether that's something we want to do.

Reply:
yes we may remove it. Currently bullshark provides sub-dags to commit